parcellab/common/templates/_job.tpl

```diff
@@ -0,0 +1,27 @@
+{{- define "common.job" -}}
+{{- $job := default (dict "enabled" false) .job -}}
+{{- if not $job.job -}}
```

Comment:
Why not just `$job`?

Reply:
Or if we want to skip the resource, then `if $job.enabled`

Reply:
fixed